scene.js

```diff
@@ -58,8 +59,8 @@ class Scene {
 
         // set up the camera position and view matrix
         var cameraPos = [-translate * Math.sin(roll) * Math.cos(-pitch),
-            -translate * Math.sin(-pitch),
-            translate * Math.cos(roll) * Math.cos(-pitch)];
+                         -translate * Math.sin(-pitch),
+                          translate * Math.cos(roll) * Math.cos(-pitch)];
```

Comment:
fair point. I have to say i prefer the code being readable and have no idea what Cesium coding style guidelines are!  i can revert these changes if that will help.